core/domain/services/graphql/types/Venue.php

```diff
@@ -229,4 +242,19 @@ public function extendMutations()
             6
         );
     }
+
+
+    /**
+     * @param EE_Venue    $venue
+     * @param array       $args
+     * @param AppContext  $context
+     * @param ResolveInfo $info
+     * @return string
+     * @throws EE_Error
+     * @throws ReflectionException
+     */
+    public function getThumbnail(EE_Venue $venue, array $args, AppContext $context, ResolveInfo $info): string
+    {
+        return get_the_post_thumbnail($venue->ID(), 'medium');
```

Comment:
Better to use the URL instead of HTML
```suggestion
        return wp_get_attachment_url(get_post_thumbnail_id($venue->ID()));
```

Reply:
but MOOOOOOMMMMMMMM!!!! All the cool kids are doing it!!!